wfe/web-front-end_test.go

```diff
@@ -76,8 +74,29 @@ eROL1ve1vmQF3kjrMPhhK2kr6qdWnTE5XlPllVSZFQenSTzj98AO
 		"n":"m5Cpx3vZ0CjATirDpbILvq78fm3Dv5RBkO1VLWFmJj5Mb54vc9oYZWc1V1k-LJoESuuPHhaNO2Eu8T9tslQWcZSzr5NImxAwMk970gVQa-Hqv-Jr6xstrBpq7TKpXHTx2FnfA2wQrfIQSlBXu0t4jdUOr3oJh-QXvma8nLITdtjpC0AZNtqd0QkRJX_90SaNrl18Rr_0JrBH9ZmUSFcf3mo_BtL0Gx0jE3n-iwCI8rQtfyVP__9-n__r4IhalKLzaeio6o-qrdemh0EZgjKGCS1_RpTIeArkO8uia1KgOq-z-GfemKEm4s07WO_a0_9dLqbvpnyyZvUi405m3vGDfQ",
 		"e":"AAEAAQ"
 	}`
+
+	RandomCert = "308201ba30820166a00302010202020675300b06092a864886f70d01010b3034" +
```

Comment:
For those following along at home, here's [the decoding](http://lapo.it/asn1js/#308201BA30820166A00302010202020675300B06092A864886F70D01010B30343110300E06035504061307417573747269613111300F060355040A1308486561706C6F636B310D300B060355040B130454484F52301E170D3039313131303233303030305A170D3139313131303233303030305A30343110300E06035504061307417573747269613111300F060355040A1308486561706C6F636B310D300B060355040B130454484F52305C300D06092A864886F70D0101010500034B003048024100A4A5007FBCDA762D88F72AC6601571D989B397A7C2FBC260247C773E0B95C09D100B43170FE06C4A649529078B90F9E611A89CA470B3513F49FBB42E555C6CD90203010001A3643062300E0603551D0F0101FF040403020084301D0603551D250416301406082B0601050507030206082B06010505070301300F0603551D130101FF040530030101FF300E0603551D0E04070405010203040530100603551D230409300780050102030405300B06092A864886F70D01010B034100212F9CADEA729D8EB36D9DFC035F2F16BD561A0B30DFA5F838C0CC71289EE5FAEED60D5C3EB1A87AECAC827855DCC2FB89DDBFE05EBCDC667D0F2E31CCF1D90D).  I would prefer if this cert were a little less random :) i.e., if it had a more innocuous set of names and extensions.  Please also add a comment saying what this cert is for and what's in it.